modules/wizard/wizard.php

```diff
@@ -365,7 +365,7 @@ public function add_step_licenses( $steps ) {
 		// Add ajax action on deactivate button in licenses step.
 		add_action( 'wp_ajax_pll_deactivate_license', array( $this, 'deactivate_license' ) );
 
-		wp_enqueue_script( 'pll_admin', plugins_url( '/js/admin' . $this->get_suffix() . '.js', POLYLANG_FILE ), array( 'jquery', 'wp-ajax-response', 'postbox', 'jquery-ui-selectmenu' ), POLYLANG_VERSION, true );
+		wp_enqueue_script( 'pll_admin', plugins_url( '/js/admin' . $this->get_suffix() . '.js', POLYLANG_FILE ), array( 'jquery' ), POLYLANG_VERSION, true );
```

Comment:
It seems this script will always be dequeued at line 466, because both methods are always executed.

Reply:
Yes see my comment in the PR description.
The second enqueue need a dependency which doesn't exist in the first one.
The only way to update a dependency on a script already enqueued is to dequeue it before.

I could put the first enqueue in the condition when we add the license step which will work with Polylang which doesn't have a license step but won't work with Polylang Pro without dequeuing.

I also could keep as before with the same dependency in the first enqueue however it's strange to say we have a dependency which isn't needed. It's mainly why I decided to do like that.

As we discussed, It's preferable in the future